apache/ports.conf.in

```diff
@@ -1,3 +1,4 @@
+Listen 80
```

Comment:
not sure about the impact but I guess it's ok to also listen to port 80, at least you can currently make requests via http that are correctly answered. On K8S it should then be only 80 (no 443 anymore in the end, ssl offloading is done on CF level, afterwards traffic is only http).

Reply:
OK, removed SSL setting

Reply:
@procrastinatio How is this done currently? Is Apache responsible for SSL checking? Then it should probably remain until migration is succesfully finished, otherwise we'll break current production..

Reply:
Good point. As soon as this is clarified, I'll merge this, so the list of open PRs will shrink a bit.

Reply:
These files on the `vhosts` are managed through `puppet`. This one is only a copy of it and the `docker` config is only mimicking the `vhosts`. All docker settings have absolutely no influence on the current project.  Currently, the SSL termination is on CF.